pom.xml

```diff
@@ -1,44 +1,57 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
     <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <groupId>de.skuzzle</groupId>
         <artifactId>skuzzle-parent</artifactId>
         <version>3.0.1</version>
+        <relativePath />
     </parent>
+
     <groupId>de.skuzzle.enforcer</groupId>
     <artifactId>restrict-imports-enforcer-rule</artifactId>
     <version>2.2.0</version>
     <packaging>jar</packaging>
+
     <name>Restrict Imports Enforcer Rule</name>
     <description>Bans imports of specified classes/packages</description>
+    <url>https://github.com/skuzzle/${github.name}</url>
+
+    <scm>
+        <connection>scm:git:git://github.com/skuzzle/${github.name}.git</connection>
+        <developerConnection>scm:git:https://github.com/skuzzle/${github.name}.git</developerConnection>
+        <url>https://github.com/skuzzle/${github.name}</url>
+        <tag>HEAD</tag>
+    </scm>
+
     <properties>
         <coveralls.skip>false</coveralls.skip>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <maven.compiler.source>1.8</maven.compiler.source>
+        <jacoco-plugin.version>0.8.9</jacoco-plugin.version>
+        <maven.compiler.target>8</maven.compiler.target>
+        <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.release>8</maven.compiler.release>
-        <version.enforcer-api>3.0.0</version.enforcer-api>
-        <version.maven>2.2.1</version.maven>
-        <version.junit>5.8.1</version.junit>
-        <version.assertj>3.13.2</version.assertj>
-        <version.equals-verifier>3.7.2</version.equals-verifier>
-        <version.slf4j>1.7.25</version.slf4j>
-        <version.mockito>4.0.0</version.mockito>
+        <maven.surefire.version>3.0.0</maven.surefire.version>
+        <maven.failsafe.version>3.0.0</maven.failsafe.version>
+        <version.enforcer-api>3.3.0</version.enforcer-api>
+        <version.maven>3.9.1</version.maven>
+        <version.junit>5.9.2</version.junit>
+        <version.assertj>3.24.2</version.assertj>
+        <version.byte-buddy>1.14.4</version.byte-buddy>
+        <version.equals-verifier>3.14.1</version.equals-verifier>
+        <version.slf4j>1.7.36</version.slf4j>
+        <version.mockito>4.11.0</version.mockito>
         <version.jimfs>1.2</version.jimfs>
-        <version.java-parser>3.24.10</version.java-parser>
-        <version.resources-plugin>3.2.0</version.resources-plugin>
+        <version.java-parser>3.25.2</version.java-parser>
+        <version.resources-plugin>3.3.1</version.resources-plugin>
         <version.invoker-plugin>3.5.1</version.invoker-plugin>
-        <version.spotless-plugin>2.34.0</version.spotless-plugin>
+        <version.spotless-plugin>2.36.0</version.spotless-plugin>
```

Comment:
I see why it doesn't build on my jenkins right now. This spotless plugin version currently doesn't work on my jenkins because it tries to access a file at a location where it has no permissions. I haven't had time to properly fix this so for now we need to stick with the slightly older version.